### PR TITLE
LA-273: Update swagger documentation

### PR DESCRIPTION
**Pull request by bpoon-selina**

### CODE REVIEW

**Comment by selina-bot-user** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **bpoon-selina**
You can assign the PR to them by writing `/assign @bpoon-selina` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=Selina-Finance%2Fms-lending-api).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/Selina-Finance/ms-lending-api/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by selina-bot-user** [comment]
:star: PR built and available in a preview **selina-finance-ms-lending-api-pr-129** [here](http://ms-lending-api-pr129.selinadev.co.uk)

File: src/main/java/com/selina/lending/internal/dto/LendingConstants.java [hunk 1]
```diff
@@ -27,16 +27,27 @@ public final class LendingConstants {
...
-    public static final String STAGE_OVERWRITE =  "DIP - Credit Commitments";
+    public static final String STAGE_OVERWRITE = "DIP - Credit Commitments";
 
     public static final String EXAMPLE_DATE = "2001-01-01";
 
     public static final String DATE_TIME_FORMAT = "yyyy-MM-dd'T'HH:mm:ss.SSSSSS";
 
     public static final String DATE_FORMAT = "yyyy-MM-dd";
 
+    public static final String INVALID_CREDENTIALS_EXAMPLE = "{\n  \"title\": \"Bad Request\",\n  \"status\": 400,\n  \"detail\": \"message details\"\n }";
```
Comment (Vadym-Lopatka) [inline src/main/java/com/selina/lending/internal/dto/LendingConstants.java:40]: Betty, can we please move it into a separate swagger error examples file?

**Review by Vadym-Lopatka — CHANGES_REQUESTED** [review]

File: src/main/java/com/selina/lending/internal/dto/LendingConstants.java (hunk 1 above)
Comment (bpoon-selina) [reply src/main/java/com/selina/lending/internal/dto/LendingConstants.java]: done

**Review by Vadym-Lopatka — APPROVED** [review]

**Comment by selina-bot-user** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[Vadym-Lopatka](https://github.com/Selina-Finance/ms-lending-api/pull/129#pullrequestreview-1299796073 "Approved")*

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=Selina-Finance%2Fms-lending-api).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/Selina-Finance/ms-lending-api/blob/main/OWNERS)~~ [Vadym-Lopatka]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by selina-bot-user** [comment]
:star: PR built and available in a preview **selina-finance-ms-lending-api-pr-129** [here](http://ms-lending-api-pr129.selinadev.co.uk)

**Pull request merged**